### PR TITLE
Allow jinja includes in swagger.yaml

### PR DESCRIPTION
**Pull request by jakul**
This PR implements the Jinja template loading mechanism specified in #254, which will let us split up our swagger.yaml into smaller files and include them at runtime.

Changes proposed in this pull request:
- Allow swagger.yaml to specify Jinja `{% include %}` blocks

### CODE REVIEW

**Comment by hjacobs** [comment]
I'm not sure whether we want to support full-blown Jinja macros, one issue might be interoperability (using the same Swagger YAML in IDE/editors).

**Comment by jakul** [comment]
Using the Jinja includes will definitely break editors etc. My preferred solution would be to use the `$ref`s as defined by the Swagger spec, but Connexion doesn't support that yet (and I'm pretty certain it's non-trivial to add).

This PR is intended a stop-gap solution until Connexion has full support for the spec.

**Comment by hjacobs** [comment]
@jakul thanks for the PR first of all :smirk:

**Comment by rafaelcaricio** [comment]
@jakul thanks for the PR.

But I am 👎  for merging this PR. My concerns:

- If we add the support for full-blown Jinja macros and people start relying on it then will be difficult to remove this support later;
- All editors will break sounds not good to me. We are going to add support to a non-standard way of writing OpenAPI specifications;
- The support for full-blown Junja will not need to be done inside Connexion code after #314 gets merged. You will be able to render the specification and then just pass it to `add_api` in Connexion.

File: docs/cookbook.rst [hunk 1]
```diff
@@ -67,3 +67,56 @@ Connexion application. Keep in mind that the format checkers should be
...
+    # properties.yaml
+    properties:
+      title:
+        type: string
+      price_label:
+        type: string
+        format: money
+
+
+This appraoch won't work if you are including block which should be indented.
```
Comment (Linkid) [inline docs/cookbook.rst]: Here is a typo: "appraoch" → "approach" ;).

File: docs/cookbook.rst [hunk 2]
```diff
@@ -67,3 +67,56 @@ Connexion application. Keep in mind that the format checkers should be
 defined and registered before you run your application server. A full
 example can be found at
 https://gist.github.com/rafaelcaricio/6e67286a522f747405a7299e6843cd93
+
+
+Splitting up your swagger.yaml
+------------------------------
+
+Connexion [does not support](https://github.com/zalando/connexion/issues/254) the `$ref` syntax from Swagger which allows you
+to split up one big swagger.yaml` into multiple smaller ones. There is a
```
Comment (Linkid) [inline docs/cookbook.rst]: Missing a "`" ?

**Comment by jakul** [comment]
Thanks for the feedback. I'm happy to use the approach in #314.

**Pull request closed without merging**